addons/sale_loyalty/models/sale_order.py

```diff
@@ -176,6 +184,21 @@ def action_open_reward_wizard(self):
             return True
         return self.env['ir.actions.actions']._for_xml_id('sale_loyalty.sale_loyalty_reward_wizard_action')
 
+    def action_view_gift_cards(self):
+        self.ensure_one()
+        return {
+            'name': 'Gift Cards',
+            'type': 'ir.actions.act_window',
+            'view_mode': 'list',
```

Comment:
don't we need to add form view :thinking:

Reply:
Yes, we can definitely add a form view as well. However, the task specifications focused on the need for easy navigation to the gift card and sending it, and the "Send" button is located on the List View. I opted to keep it limited to that for now. Nonetheless, I will add the form view too, as it would be a nice addition.